src/scripts/windows/install.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+Install_AWS_CLI(){
+    if [ "$1" = "latest" ]; then
+        version=""
+    else
+        version="$1"
+    fi
+    echo "Installing AWS CLI v2"
+    cd /tmp || exit
+    
+    if [ ! "$(command -v choco)" ]; then
+        echo "Chocolatey is required to uninstall AWS"
```

Comment:
Was this supposed to be `install` instead of `uninstall`?

```suggestion
        echo "Chocolatey is required to install AWS"
```